src/Content/Content.js

```diff
@@ -1,14 +1,63 @@
-import React from 'react';
-import  './content.scss';
+import './content.scss';
 
 
+const Greeting = ({ name }) => {
+  const hour = new Date().getHours();
+  let partDay = '';
+  if (hour > 22.00 && hour < 3.00) {
```

Comment:
1) `new Date().getHours()` return integer number, do not use point here (just use _22_, _3_ etc)
2) in what case variable _hour_ can be more than 22 and less than 3 the same time? If it is 23 and higher, it can't be less then 3. Fix this.